docs/Tutorials/Basics/getting-started.md

```diff
@@ -0,0 +1,146 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'; 
+
+# Getting Started
+
+In the basic tutorials, you will be shwon how to create [the four types](/Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/introduction.md) of Keypom drops. Two methods will be shown.
+
+1) [NEAR-API-JS](https://docs.near.org/tools/near-api-js/reference)  
+2) [Keypom-JS SDK](https://github.com/keypom/keypom-js) <-- currently linked to github, change this to typedocs or smth else once ready)
+
+In the Deploy scripts, the same four types of Keypom drops will be created, but you will be able to customize the configurations to your needs. 
+
+---
+
+## Prerequisites
+For the basic tutorials, you can choose to run the scripts on your own machine. To do son, you must have the following:
+
+1. [Node JS](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)  
+2. [NEAR-API-JS](https://docs.near.org/tools/near-api-js/quick-reference#install)  
+3. To install the SDK, simply run the following in your command prompt.  
+```bash
+npm -i keypom-js
+```
+
+At this point, you are ready to start the tutorials. To interact with the deploy scripts, you can continue reading this page.  
+
+For the Deploy Scripts, you should have a [NEAR account](https://docs.near.org/concepts/basics/account). It can be either `testnet` or `mainnet`.
+:::note
+If you choose to use mainnet, you will be using real $NEAR to fund your drops.
+:::
+
+---
+
+## Deploy Scripts
```

Comment:
we can put any deploy script stuff in its own category if you want. That way we create the separation b/n the tutorials and the deploy scripts.

Reply:
Yes because docs shouldn't really be referencing the deploy scripts, those belong to the protocol and so we should add this documentation that you have into the readme.